integration_tests/utils.js

```diff
@@ -158,19 +158,12 @@ const extractSummary = (stdout: string) => {
 // unifies their output to make it possible to snapshot them.
 const cleanupStackTrace = (output: string) => {
   return output
-    .replace(/\n.*at.*timers\.js.*$/gm, '')
-    .replace(/\n.*at.*assert\.js.*$/gm, '')
-    .replace(/\n.*at.*node\.js.*$/gm, '')
     .replace(/\n.*at.*next_tick\.js.*$/gm, '')
```

Comment:
Looks like this needs to be updated on Node 4? Not sure why it fails now though.

Reply:
`next_tick` is not under `internal/` on node 4...

Reply:
It's not the integration test which fails, it's the new test I added in message-utils